src/app/pages/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,13 @@
+<app-hill-background cover></app-hill-background>
+<app-back-button top-left routerLink="/"></app-back-button>
+<div class="centered-container">
+  <div>
+    <mat-card>
+      <mat-card-title class="title">Whoops</mat-card-title>
+      <mat-card-content>
+        This page doesn't seem to exist.
```

Comment:
I think we should mention the route that's being attempted here. People using the app added to their home screen won't be able to see what route caused the error otherwise.